incubator/consul/values.yaml

```diff
@@ -0,0 +1,208 @@
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+## Kubernetes service type
+serviceType: ClusterIP
+
+## Consul replicas
+replicas: 3
+
+## Consul service ports
+httpPort: 8500
+rpcPort: 8400
+serflanPort: 8301
+serverPort: 8300
+consulDnsPort: 8600
+
+## Datacenter name for consul. If not supplied, will use the consul
+## default 'dc1'
+datacenterName: dc1
+
+## Predefined value for gossip key.
+## The key must be 16-bytes, can be generated with $(consul keygen)
+# gossipKey: 887Syd/BOvbtvRAKviazMg==
+
+## Use TLS to verify the authenticity of servers and clients.
+## Check README for more information.
+# tlsEncryptionSecretName: your-already-created-secret
+
+## Extra configuration that will be added to the default one.
+#localConfig: |-
+#  {
+#    "key": "value"
+#  }
+
+## Consul domain name.
+domain: consul
+
+## Consul address in which will bind client interfaces.
+clientAddress: '0.0.0.0'
+
+##Consul Serf lan address.
+serflanAddress: '0.0.0.0'
+
+## Consul raft multiplier.
+raftMultiplier: '1'
+
+## Consul data Persistent Volume Storage Class
+## If defined, storageClassName: <storageClass>
+## If set to "-", storageClassName: "", which disables dynamic provisioning
+## If undefined (the default) or set to null, no storageClassName spec is
+##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+##   GKE, AWS & OpenStack)
+##
+persistence:
+  enabled: true
+  # storageClass: "-"
+  accessModes:
+    - ReadWriteOnce
+  size: 8Gi
+  annotations: {}
+
+resources: {}
+#  requests:
+#    memory: 256Mi
+#    cpu: 100m
+
+## Setting maxUnavailable will create a pod disruption budget that will prevent
+## voluntarty cluster administration from taking down too many consul pods. If
+## you set maxUnavailable, you should set it to ceil((n/2) - 1), where
+## n = Replicas. For example, if you have 5 or 6 Replicas, you'll want to set
+## maxUnavailable = 2. If you are using the default of 3 Replicas, you'll want
+## to set maxUnavailable to 1.
+maxUnavailable: 1
+
+## nodeAffinity settings
+# nodeAffinity:
+#   requiredDuringSchedulingIgnoredDuringExecution:
+#     nodeSelectorTerms:
+#     - matchExpressions:
+#       - key: cloud.google.com/gke-preemptible
+#         operator: NotIn
+#         values:
+#         - true
+
+## Anti-Affinity setting. The default "hard" will use pod anti-affinity that is
+## requiredDuringSchedulingIgnoredDuringExecution to ensure 2 services don't
+## end up on the same node. Setting this to "soft" will use
+## preferredDuringSchedulingIgnoredDuringExecution. If set to anything else,
+## no anti-affinity rules will be configured.
+antiAffinity: "soft"
+
+
+## Create dedicated UI service
+##
+uiService:
+  enabled: true
+  type: "ClusterIP"
+
+## Configure the ingress resource that allows you to access the
+## Consul user interface. Set up the URL
+## ref: http://kubernetes.io/docs/user-guide/ingress/
+##
+uiIngress:
```

Comment:
should we be adding a `ui` key for the `service` and `ingress` parameters?

f.e.

```yaml
ui:
  service:
    ...
    ...
  ingress:
    ...
    ...
```